src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/images/native_otlp_with_collector.svg

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
image looks good in both light and dark mode, doesn't bleed into the right side. Your alt-text does read off its image title, so consider changing that to be more meaningful.

Reply:
@kaileyhaynes, two in a row! Thanks for finding this title issue! I just fixed it.